README.md

```diff
@@ -286,3 +286,12 @@ Push a git tag with the scheme `vX.Y.Z` (semver).
 This project is BSD 3-Clause licensed (see LICENSE file for details).
 
 This project uses other OpenSource software as listed in `go.mod` and indirectly in `go.sum` files. All their licenses apply too.
```

Comment:
The "License" section could be removed from the README. We have a dedicated LICENSE file in the file system anyway, don't we?

Otherwise, the "License" section should probably be the last section in the README, hence you'd need to move "Environment variables" up somewhere.

Reply:
jep